### PR TITLE
Support for Selenium nodes with Opera browser

### PR DESCRIPTION
**Pull request by ehbello**
### Description
This pull request provides the following changes:

1) Add the necessary files to build all types of Selenium nodes with the Opera browser. 
2) Enable existing tests for the resulting Docker Images
3) Update the README.md file to describe the usage of the 

### Motivation and Context
These changes will allow us to use Opera as a new browser to test with Selenium.

We, at [System73](https://system73.com), have been using Selenium to test our software for several years. We needed to implement these changes for Selenium nodes to work with Opera and now we are sharing them with the community.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist
- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,7 +332,7 @@ $ docker run -d -e HUB_HOST=<hub_ip|hub_name> -e NODE_MAX_INSTANCES=5 -e NODE_MA
 
 ### Running in Headless mode
 
-Both [Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Headless_mode) and [Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome) support running tests in headless mode.
+Both [Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Headless_mode), [Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome) and [Opera](https://forums.opera.com/topic/20375/opera-cli-switches-and-headless) support running tests in headless mode.
```
Comment (diemol) [inline README.md]:
```suggestion
[Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Headless_mode), [Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome) and [Opera](https://forums.opera.com/topic/20375/opera-cli-switches-and-headless) support running tests in headless mode.
```

File: README.md [hunk 2]
```diff
@@ -512,4 +578,4 @@ or
 
 `Message: unknown error: Chrome failed to start: exited abnormally`
 
-The reason _might_ be that you've set the `START_XVFB` environment variable to "false", but forgot to actually run Firefox or Chrome (respectively) in headless mode.
+The reason _might_ be that you've set the `START_XVFB` environment variable to "false", but forgot to actually run Firefox, opera or Opera (respectively) in headless mode.
```
Comment (diemol) [inline README.md]:
```suggestion
The reason _might_ be that you've set the `START_XVFB` environment variable to "false", but forgot to actually run Firefox, Chrome or Opera (respectively) in headless mode.
```

File: Makefile [hunk 3]
```diff
@@ -8,19 +8,23 @@ MAJOR := $(word 1,$(subst ., ,$(VERSION)))
 MINOR := $(word 2,$(subst ., ,$(VERSION)))
 MAJOR_MINOR_PATCH := $(word 1,$(subst -, ,$(VERSION)))
 
-all: hub chrome firefox chrome_debug firefox_debug standalone_chrome standalone_firefox standalone_chrome_debug standalone_firefox_debug
+all: hub chrome firefox opera chrome_debug firefox_debug opera_debug standalone_chrome standalone_firefox generate_standalone_opera standalone_chrome_debug standalone_firefox_debug generate_standalone_opera_debug
```
Comment (diemol) [inline Makefile]: Seems the standalone and standalone debug images are not being generated, can you please double check the Makefile?

Comment (ehbello) [reply Makefile]: Done.

File: README.md (hunk 2 above)
Comment (ehbello) [reply README.md]: Done.

**Comment by ehbello** [comment]
I removed all automatically generated files (Dockerfile, README.md, etc) at release.

**Comment by maximehyh** [comment]
Hi @ehbello, 
Thanks for all the work on the FF video support and Opera browser. 
Since it is now Chromium based I was wondering if you or @diemol had some ambition to make the same with Edge (https://www.microsoftedgeinsider.com/fr-fr/download)? 
Thx!
Maxime

**Comment by maximehyh** [comment]
Hi @ehbello,
Thanks for all the work on the FF video support and Opera browser.
Since it is now Chromium based I was wondering if you or @diemol had some ambition to make the same with Edge (https://www.microsoftedgeinsider.com/fr-fr/download)?
Thx!
Maxime

**Comment by cben** [comment]
> I removed all automatically generated files (Dockerfile, README.md, etc) at release.

FWIW, the other directories all include them in git.
- The READMEs are useful when navigating the github repo e.g. https://github.com/SeleniumHQ/docker-selenium/tree/master/NodeChrome

**Comment by ehbello** [comment]
@cben These files will be generated at release. Wait for it :wink:

**Pull request merged**